README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
The installation steps rely on Homebrew without checking if it is installed. Provide a pre-check for Homebrew installation or instructions on how to install it to avoid potential confusion for users who may not have it.